CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/tree/SpanningTreePlugin.java

```diff
@@ -41,7 +41,6 @@ public void edit(final GraphWriteMethods wg, final PluginInteraction interaction
         } else {
             rootVxId = wg.getVertex(0);
         }
-
-        final GraphWriteMethods tree = st.createSpanningTree(true, true, rootVxId);
+         st.createSpanningTree(true, true, rootVxId);
```

Comment:
This line is still required because it creates the spanningTree for this plugin but it doesn't need to be stored because the actual tree is never modified in this class